pt/news/_posts/2018-12-25-ruby-2-6-0-released.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: news_post
+title: "Ruby 2.6.0 Released"
+author: "naruse"
+translator: "vnbrs (Vinicius Brasil)"
+date: 2018-12-25 00:00:00 +0000
+lang: pt
+---
+
+Nós temos o prazer de anunciar o lançamento do Ruby 2.6.0.
+
+Ele introduz múltiplas novas funcionalidades e melhorias de performance, sendo as mais notáveis:
+
+ * Um novo compilador JIT.
+ * O módulo `RubyVM::AbstractSyntaxTree`.
+
+## JIT [Experimental]
+
+O Ruby 2.6 introduz uma implementação inicial de um compilador JIT (Just-In-Time).
+
+O compilador JIT foca em melhorar a performance das aplicações Ruby. Diferente dos compiladores JIT tradicionais, que operam no processo, o compilador JIT do Ruby escreve código C no disco e invoca um compilador comum C para gerar código nativo. Para mais detalhes sobre isso, acesse a [organização MJIT por Vladimir Makarov](https://bugs.ruby-lang.org/projects/ruby/wiki/MJIT#MJIT-organization).
+
+Para habilitar o compilador JIT, especifique `--jit` na linha de comando ou na variável de ambiente `$RUBYOPT`. Especificando `--jit-verbose=1` irá fazer com o que compilador JIT imprima informações adicionais. Leia o output de `ruby --help` ou [a documentação](https://bugs.ruby-lang.org/projects/ruby/wiki/MJIT#Basic-usage) para outras opções.
```

Comment:
It's better to use `saída` instead `output`.

Reply:
```suggestion
Para habilitar o compilador JIT, especifique `--jit` na linha de comando ou na variável de ambiente `$RUBYOPT`. Especificando `--jit-verbose=1` irá fazer com o que compilador JIT imprima informações adicionais. Leia a saída de `ruby --help` ou [a documentação](https://bugs.ruby-lang.org/projects/ruby/wiki/MJIT#Basic-usage) para outras opções.
```